### PR TITLE
Finished last fixings over errors from stickler

### PR DESCRIPTION
**Pull request by maosan132**
Finished last fixings over errors from stickler

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Within this proyect, I will construct a clone of old homepage from Apple Inc. Re
 
   - This project goes by solo.
```
Comment (euqueme) [inline README.md]: put your contact info, name, email, twitter etc...

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Queme P. (TSE)

### Hello @maosan132  :handshake: 

### Thank you for doing the previous reviewers changes :medal_sports: Your project it's looking better and better every time :art: there's a couple of things to work on :hammer: 

## Pull Request Flow

- [ ]  For future submissions please use the same pull request so after you are done with the following changes **do not :no_good_woman:  close this pull request.** just request for a code review using this one #13  so the next reviewer can see what it's been asked and the revision flow gets maintained.

## Reset Body

- [ ] The body has 8px of margin which is causing the main background to shift 8px to the right, fix this by adding a [CSS reset](https://meyerweb.com/eric/tools/css/reset/) to your CSS file or set the `body { margin: 0; }` if you do add the reset remember to add it in the **Acknowledgments** section in your **Readme** file (Don't have one? create it! :wrench: ).

## Horizontal Scroll
- [ ] also, the bottom images are too wide and are generating a horizontal scroll, take into account margin and padding sizes as they might push away the objects resulting in needing more screen width 

image

## Building with Backgrounds

- [ ] [OPTIONAL] This project is all about building with backgrounds and gradients so please change the `<aside>` images to be `<div>` tags with background images.

[OPTIONAL] please create a Live Demo of your project and add the link to your Readme file you can follow these [steps](https://help.github.com/en/github/working-with-github-pages/creating-a-github-pages-site#creating-your-site).

I have made some comments in your Readme File please reply and resolve them  :arrow_down_small: 

### That's it! :raised_hands: 

### Please :pray:  do the changes :memo:  and submit back for a code review when you are done :rocket: 

Happy coding! :woman_technologist: 

any questions? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

**Comment by maosan132** [comment]
Already changed the aside ul for divs, and set backgrounds for all of them. I do have the margin of the body tag set to 0, something with the browser puts the 8px to it. Horizontal scrolls were elimintated.

**Comment by euqueme** [comment]
> Already changed the aside ul for divs, and set backgrounds for all of them. I do have the margin of the body tag set to 0, something with the browser puts the 8px to it. Horizontal scrolls were elimintated.

Thank you for your explanation, it will be noticed by the next reviewer :raised_hands:

File: README.md (hunk 1 above)
Comment (maosan132) [reply README.md]: Changes made, everything is up to the suggested changes.

**Review by nikasulo — APPROVED** [review]
# Such great work! @maosan132 
Great work incorporating the changes requested by the previous TSE. I commend your hard work and attention to detail. That being said, please merge this and carry on! 

#### JUST A TIP: 
You can reset all the margin and padding on HTML elements using this wildcard `*{margin: 0; padding: 0;}` in your CSS file.

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](.https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can. My available times on social media are 6-7pm and 10-11pm(WAT).

**Pull request merged**